doc-surrealdb_versioned_docs/version-1.x/surrealql/functions/script/arguments.mdx

```diff
@@ -11,12 +11,12 @@ Additional arguments can be passed in to the function from SurrealDB, and these
 
 ```surql 
 -- Create a new parameter
-LET $value = "SurrealDB";
+LET $val = "SurrealDB";
```

Comment:
nitpick: I would prefer `$value` still

Reply:
Apparently the predefined variables might be outright restricted one day, so thought I'd get a jump on this one just in case. (Not a huge deal either way for something this small but might as well)

Reply:
Indeed, I read this code quite some time ago. I thought it was already active by now. Still prefer a clear variable name like `$name` but not a big deal anyway.